src/Pos/Block/Logic.hs

```diff
@@ -751,16 +750,18 @@ createMainBlockPure
     -> SscPayload ssc
     -> UpdatePayload
     -> SecretKey
-    -> MainBlock ssc
+    -> m (MainBlock ssc)
 createMainBlockPure limit prevHeader txs pSk sId psks sscData usPayload sk =
-    flip evalState limit $ do
+    flip evalStateT limit $ do
```

Comment:
Well, this is thought to be `usingStateT` but @gromakovsky raised reasonable concern that semantic of `usingStateT` is not clear. So `usintStateT` returns pair and there're `evaluatingStateT` and `executingStateT` for consistency. Not to write shorter code but for people who don't like `flip` :D Also, in some perfect future with autocompletion IDE features, it should require less effort from programmer to write `evaluatingStateT` than `flip evalStateT`.